src/app/static/src/app/store/load/actions.ts

```diff
@@ -102,8 +102,22 @@ export const actions: ActionTree<LoadState, RootState> & LoadActions = {
         //storage then reload the page, to follow exactly the same fetch and reload procedure as session page refresh
         //NB load state is not included in the saved state, so we will default back to NotLoading on page reload.
 
-        // Backwards compatibility fix: projects which calibrated before bug fix in mrc-3126 have empty calibrate options
+
         const {modelCalibrate} = savedState
+
+        // Backward compatibility
```

Comment:
As with the other backward compatibility comment, it might be nice to reference the ticket in case it's not obvious what the relevant change was:
```suggestion
        // Backward compatibility fix: projects which calibrated before mrc-4538 have metadata embedded in result
```